src/rich-text-editor.component.js

```diff
@@ -228,8 +228,10 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   }
 
   function getBrowserDefaultStyling() {
-    if (props.placeholderColor) {
-      return `color: ${props.placeholderColor};`
+    if (props.placeholderStyle) {
+      return Object.entries(props.placeholderStyle).reduce((styleString, [propName, propValue]) => {
+        return `${styleString}${propName.replace(/[A-Z]/g, match => `-${match.toLocaleLowerCase()}`)}:${propValue};`
+      }, '')
```

Comment:
clever move [here](https://stackoverflow.com/questions/45205593/how-to-convert-a-json-style-object-to-a-css-string), I think this covers all cases?